app/models.py

```diff
@@ -1431,6 +1457,18 @@ def validates_data(self, key, data):
 
         return data
 
+    @validates('awarded_at')
+    def validates_awarded_at(self, key, awarded_at):
+        if self.awarded_at is not None:
+            raise ValidationError('Brief response award cannot be changed as the brief has already been awarded.')
```

Comment:
This error message could be a little clearer either to say that you can not award this brief response more than once or that this brief response has already been awarded